app/src/main/java/net/gsantner/markor/frontend/textview/SyntaxHighlighterBase.java

```diff
@@ -328,7 +328,7 @@ public final synchronized SyntaxHighlighterBase reflow(final int[] range) {
      *
      * @return this
      */
-    public synchronized final SyntaxHighlighterBase recompute() {
```

Comment:
We do everything in the UI thread anyway (perhaps we shouldn't) and 'synchronized' seems to introduce some overhead